Assets/Scripts/Audio/AudioBuilderSystem.cs

```diff
@@ -15,9 +16,15 @@ public class AudioBuilderSystem : MonoBehaviour
 
         private void Awake()
         {
-            _builtClips = new List<AudioClip>();
+            _builtClips = new List<AudioClip>(new AudioClip[10]);
```

Comment:
Support for up to 10 stage positions (aka audio files) built at once. Seems plenty to me? 
I could remove this magic number...

Reply:
Lets just expose this number just in case, no harm it being a variable. Default to 10 is plenty i reckon.

Reply:
I will do it now and then merge the changes